<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>NewBorrowDialog</class>
    <widget class="QDialog" name="NewBorrowDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>400</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Borrow New Book</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="text">
                        <string>Select a Book to Borrow</string>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>14</pointsize>
                            <weight>75</weight>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer1">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>10</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QLabel" name="bookListLabel">
                    <property name="text">
                        <string>Available Books:</string>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>10</pointsize>
                            <weight>75</weight>
                            <bold>true</bold>
                        </font>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QListWidget" name="bookListWidget">
                    <property name="alternatingRowColors">
                        <bool>true</bool>
                    </property>
                    <property name="selectionMode">
                        <enum>QAbstractItemView::SingleSelection</enum>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer2">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>10</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QHBoxLayout" name="dateLayout">
                    <item>
                        <widget class="QLabel" name="returnDateLabel">
                            <property name="text">
                                <string>Return Date:</string>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>10</pointsize>
                                    <weight>75</weight>
                                    <bold>true</bold>
                                </font>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QDateEdit" name="returnDateEdit">
                            <property name="calendarPopup">
                                <bool>true</bool>
                            </property>
                            <property name="displayFormat">
                                <string>dd/MM/yyyy</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <spacer name="verticalSpacer3">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>